### PR TITLE
(#5251) - Remove new PouchDB().then

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by daleharvey** [comment]
Pretty happy with how much cleaner / easier to follow `constructor.js` is now

**Comment by nolanlawson** [comment]
I will try to review this soon.

File: packages/pouchdb-core/src/constructor.js [hunk 1]
```diff
@@ -46,135 +38,82 @@ function prepareForDestruction(self, opts) {
...
-      return self.constructor.sync(self, dbName, opts, callback);
-    };
+  self.replicate.to = function (url, opts, callback) {
+    return self.constructor.replicate(self, url, opts, callback);
+  };
 
-    self.replicate.sync = self.sync;
+  self.sync = function (dbName, opts, callback) {
+    return self.constructor.sync(self, dbName, opts, callback);
+  };
```
Comment (nolanlawson) [inline packages/pouchdb-core/src/constructor.js]: we can move these into the replication module

**Comment by nolanlawson** [comment]
Great work, this is awesome!

**Pull request merged**